internal/ce_client/ce_client_test.go

```diff
@@ -6,13 +6,9 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func IntMin(a, b int) int {
-	if a < b {
-		return a
-	}
-	return b
-}
-
-func TestIntMinBasic(t *testing.T) {
-	assert.Equal(t, -2, IntMin(2, -2))
+func TestNewCeClient(t *testing.T) {
+	_, err := NewCeClient("podman")
```

Comment:
No it will not fail if podman is not installed. At the moment, I think the point is moot because when you pass in `podman` as an argument, you won't get past [this error](https://github.com/mfleader/arcaflow-plugin-image-builder/blob/75ac5976afcb65ccf2f2aaf412c9dc5a9e3fad5f/internal/ce_client/ce_client.go#L20).

Reply:
I changed the one that carpenter interacts with to `ContainerEngineService`.